### PR TITLE
prepare release

### PR DESCRIPTION
**Pull request by ostrya**
- update notice file
- add version information to standalone

Signed-off-by: Kai Helbig <kai.helbig@tngtech.com>

### CODE REVIEW

File: standalone/build.gradle [hunk 1]
```diff
@@ -63,3 +64,9 @@ configurations {
 application {
     mainClassName = 'com.tngtech.keycloakmock.standalone.Main'
 }
+
+buildConfig {
+    buildConfigField('String', 'NAME', "\"${project.name}\"")
+    buildConfigField('String', 'VERSION', "\"${project.version}\"")
+    buildConfigField('long', 'BUILD_TIME', "${System.currentTimeMillis()}L")
```
Comment (ostrya) [inline standalone/build.gradle:71]: Hm ... might be nice to have, but it's currently unused. What do you think?

Comment (ripssi) [reply standalone/build.gradle:71]: you mean only the build time or all three lines?

Comment (ostrya) [reply standalone/build.gradle:71]: Only the build time. The other two are used in the main class of the standalone server.

Comment (ripssi) [reply standalone/build.gradle:71]: well ... we can add it when we find a place to use it ;-)

Comment (ostrya) [reply standalone/build.gradle:71]: So you mean to say:
```suggestion
```

**Pull request merged**